GitUI/Plugin/PluginRegistry.cs

```diff
@@ -27,29 +27,31 @@ public static void Initialize()
                     return;
                 }
 
-                try
-                {
-                    ManagedExtensibility.SetUserPluginsPath(AppSettings.UserPluginsPath);
-
-                    foreach (var plugin in ManagedExtensibility.GetExports<IGitPlugin>().Select(lazy => lazy.Value))
-                    {
-                        Validates.NotNull(plugin.Description);
-
-                        // Description for old plugin setting processing as key
-                        plugin.SettingsContainer = new GitPluginSettingsContainer(plugin.Id, plugin.Description);
-
-                        if (plugin is IRepositoryHostPlugin repositoryHostPlugin)
-                        {
-                            GitHosters.Add(repositoryHostPlugin);
-                        }
+                return;
 
-                        Plugins.Add(plugin);
-                    }
-                }
-                catch
-                {
-                    // no-op
-                }
+                ////try
+                ////{
+                ////    ManagedExtensibility.SetUserPluginsPath(AppSettings.UserPluginsPath);
+
+                ////    foreach (var plugin in ManagedExtensibility.GetExports<IGitPlugin>().Select(lazy => lazy.Value))
+                ////    {
+                ////        Validates.NotNull(plugin.Description);
+
+                ////        // Description for old plugin setting processing as key
+                ////        plugin.SettingsContainer = new GitPluginSettingsContainer(plugin.Id, plugin.Description);
+
+                ////        if (plugin is IRepositoryHostPlugin repositoryHostPlugin)
+                ////        {
+                ////            GitHosters.Add(repositoryHostPlugin);
+                ////        }
+
+                ////        Plugins.Add(plugin);
+                ////    }
+                ////}
+                ////catch
+                ////{
+                ////    // no-op
+                ////}
```

Comment:
❔ Does this disable all extensions functionality, or just a subset? Can we link it to a follow-up issue?

Reply:
Plugin loading is currently disabled because I couldn't get it working. Partially because the build props and targets need to be reviewed and rewritten for `dotnet build` instead of the dotnet/arcade scripts. 
This is will be a follow up work